docs/tutorials/welcome.md

```diff
@@ -28,9 +28,9 @@ externs have been generated.
 You can build Shaka on Linux, Windows, or Mac.
 To get the sources and compile the library, you will need:
   * {@link https://git-scm.com/downloads Git v1.9+}
-  * {@link https://www.python.org/downloads/ Python v2.7 or v3.5+}
-  * {@link https://learn.microsoft.com/en-us/java/openjdk/download Java Runtime Environment v14+}
-  * {@link https://nodejs.org/en/download/ NodeJS v14+}
+  * {@link https://www.python.org/downloads/ Python v3.5+}
+  * {@link https://learn.microsoft.com/en-us/java/openjdk/download Java Runtime Environment v21+}
+  * {@link https://nodejs.org/en/download/ NodeJS v18+}
```

Comment:
Is there a specific reason to use Node 18 instead of 24? 18 is already passed EOL, and 20 is in maintenance mode. 

https://nodejs.org/en/about/previous-releases

Reply:
does karma work as expected in latest LTS?